test/run_readelf_tests.py

```diff
@@ -142,13 +145,27 @@ def prepare_lines(s):
         return False, 'Number of lines different: %s vs %s' % (
                 len(lines1), len(lines2))
 
+    view_col_position = None
```

Comment:
Can you add a comment saying what the type here is?

Reply:
Nullable integer :) It's a position within string.